src/_sass/base/_base.scss

```diff
@@ -265,3 +265,17 @@ blockquote {
     text-align: center;
   }
 }
+
+summary {
+  margin-bottom: 1rem;
+}
+
+ol + img, ul + img, ol + p, ul + p, p + p + img {
+  margin-bottom: 1rem;
+}
+
+.figure-caption {
+  font-size: .875rem;
+  font-style: italic;
+  color: #041E3C;
```

Comment:
Is there a reason to use this over the normal text color `$site-color-body`?

Reply:
I wanted it to stand out a bit more. I've commented out most of the images, though, so this may be vestigal.